doc/tutorials/12-constant_pH/12-constant_pH.ipynb

```diff
@@ -170,7 +170,9 @@
     "\n",
     "# dependent parameters\n",
     "Box_V = (N_acid/N_A/c_acid)\n",
-    "Box_L = np.cbrt(Box_V.to('m**3'))*ureg('m')\n",
+    "Box_L = np.cbrt(Box_V.to('m**3'))\n",
```

Comment:
shouldn't you explicitly check for the breaking pint version. We will probably not remember why the check for a missing unit is necessary.

Reply:
something like this, instead of the isinstance check?
```python3
if tuple(map(int, pint.__version__.split('.'))) >= (0, 10):
    Box_L *= ureg('m')
```